jerry-core/api/jerry-debugger.c

```diff
@@ -145,6 +149,13 @@ jerry_debugger_wait_and_run_client_source (jerry_value_t *return_value) /**< [ou
           break;
         }
 
+        /* Stop waiting for a new source file. */
+        if ((JERRY_CONTEXT (debugger_flags) & JERRY_DEBUGGER_CLIENT_SOURCE_CLOSED))
+        {
+          ret_type = JERRY_DEBUGGER_SOURCE_CLOSED;
```

Comment:
JERRY_DEBUGGER_SOURCE_CLOSED ???? The comment says JERRY_DEBUGGER_NO_MORE_SOURCE.